geometry_msgs/Pose2D goal
std_msgs/String id
---
std_msgs/String response